e2e/test/scenarios/dashboard-filters/old-parameters.cy.spec.js

```diff
@@ -117,6 +123,41 @@ describe("scenarios > dashboard > OLD parameters", () => {
       saveDashboard();
       cy.button("Apply").should("not.exist");
     });
+
+    it("should not clear existing parameter values after editing dashboards", () => {
+      cy.get("@dashboardId").then(dashboardId => {
+        visitDashboard(dashboardId, {
+          params: {
+            category: "Gadget",
+          },
+        });
```

Comment:
I move `visitDashboard` inside each test because I want to visit different URL, that's also because I want to avoid having to add parameters value via the UI that could slow the E2E test down.